UPGRADE.md

```diff
@@ -0,0 +1,37 @@
+# Upgrade from 0.18 to 0.19
+
+## Depfile (Configuration File)
+
+### What?
+
+* The location of the default configuration file has changed.
+
+    ```
+    From: ./depfile.yaml
+    To:   ./deptrac.yaml
+    ```
+
+* The configuration inside the file must now be nested under `parameters:`
+  except for imports. See docs for examples.
+
+### How?
+
+In your `depfile.yaml` you can just add a new section `parameters:` at the top
+and then indent the remaining config 1 level under this section. The only
```

Comment:
From my understanding, this means that if someone had existing parameters with the key/name of the internal "parameters" of Deptrac itself that we have now moved there, there is going to be a conflict. Should we mention this anywhere?

Reply:
Yes, I guess but I don't think that's a big issue. I would like to keep the guide short and I don't think that is too common anyway